resolver/src/main/java/io/netty/resolver/ResolvedAddressTypes.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.resolver;
+
+/**
+ * Defined resolved address types.
+ * Ideally, we should be using "InternetProtocolFamily"
+ * but this class sits in the "transport" module so it can't be used here.
+ */
+public enum ResolvedAddressTypes {
```

Comment:
The plural sounds better to me, as it's a kind of strategy. Maybe a native english speaker would know better? @Scottmitch ?